scripts/run-unit-tests.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+# Really should not rely on PWD here
+SOURCE_DIR=${SOURCE_DIR:-$PWD}
```

Comment:
I'm not really sure what the intended run is for this, it seems to assume it is run from a particular location but it's not documented